Project.toml

```diff
@@ -1,10 +1,10 @@
 name = "AWSTools"
 uuid = "83bcdc74-1232-581c-948a-f29122bf8259"
 authors = ["Invenia Technical Computing"]
-version = "1.10.0"
+version = "2.0.0"
```

Comment:
Is it at all possible to do this without a breaking release?

Reply:
We're switching to a different `AWSConfig` type here so it seems best to go with a clean break to avoid confusion. Otherwise we might cause some odd errors down the line.

Reply:
This package is in a very odd state to begin with. So multiple major releases are warranted are fine, IMO. I plan on archiving it at one point anyways.

Reply:
This blocks so many of our internal packages; when ever a breaking release is made here.
Maybe we should look into dropping it in a bunch of places.
I bet we don't even use it.

Reply:
The move to internally using `AWS.jl` is already resulting in changes needed to be made.

Reply:
We were going to keep 2.0.0 here?